components/status-message/src/StatusMessage.tsx

```diff
@@ -0,0 +1,47 @@
+import { FC, createElement as h } from 'react';
+import { StandardProps, classBuilder } from '@not-govuk/component-helpers';
+import { AnchorList } from "@not-govuk/anchor-list";
+
+import '../assets/StatusMessage.scss';
+
+export type StatusAction = {
+  /** Location to link to */
+  href: string
+  /** Text of the link */
+  text: string
+  /** Title of the link */
+  title?: string
+};
+
+export type StatusMessageProps = StandardProps & {
+  /** Summary of status */
+  status: string,
+  /** Extra information about the status */
+  children?: any,
+  /** Links to perform status related actions */
+  actions?: StatusAction[]
+};
+
+export const StatusMessage: FC<StatusMessageProps> = ({
+    classBlock,
+    classModifiers,
+    className,
+    status,
+    children,
+    actions,
+    ...attrs
+  }) => {
+  const classes = classBuilder('hods-status-message', classBlock, classModifiers, className);
+
+  return status || children ? (
+    <div {...attrs} className={classes()}>
```

Comment:
I suppose another thing to consider would be whether you could use a Panel here: https://not-gov.uk/components?name=Panel

But that uses an `<h1>` for the heading which doesn't seem to be what you want.

Maybe in the future we could make Panel more flexible. (A `headerAs="p"` prop or something.)

Reply:
That's a good shout actually. It seems there are quite a few components that are very similar. The [stop and think pattern](https://design.homeoffice.gov.uk/patterns/stop-and-think) could probably also be created from one of these components.
I think I'll look at this as a separate PR though!

Reply:
Remember Karwai is dropping some of those patterns.

I'd check the stuff she's actually migrated over. And also give her a shout.